### DISCUSSION TITLE
[Closed] Add Extension Methods for ItemsView

### DISCUSSION THREAD

**Discussion by TheCodeTraveler** [CommunityToolkit/Maui.Markup#50] [New Feature Discussions]
This proposal will add support for `CollectionView` and `CarouselView` which both implement `ItemsView`.

I propose that we add extension methods for all of the `ItemsView` properties:
- `EmptyView`
- `EmptyViewTemplate`
- `ItemSource`
- `HorizontalScrollBarVisibility`
- `VerticalScrollBarVisibility`
- `RemainingItemsThreshold`
- `RemainingItemsReachedThresholdCommand`
- `RemainingItemsReachedThresholdCommandParameter`
- `ItemTemplate`
- `ItemsUpdatingScrollMode`

**Comment by pictos** [comment]
I approve this feature ✅

**Reply by TheCodeTraveler** [reply]
Promoted to Proposal: https://github.com/CommunityToolkit/Maui.Markup/issues/51

**Comment by CliffAgius** [comment]
Closed and answered
